tfrecord/reader.py

```diff
@@ -325,8 +364,8 @@ def tfrecord_loader(data_path: str,
         instance of a `SequenceExample`.
     """
     if sequence_description is not None:
-        return sequence_loader(data_path, index_path, description, sequence_description, shard)
-    return example_loader(data_path, index_path, description, shard)
+        return sequence_loader(data_path, index_path, description, sequence_description, shard, compression_type)
```

Comment:
Seems like there was a bug here. The order of shard and sequence_description is wrong. While you are at it could you fix this. Even better could you pass all the values by keyword argument.